relentless/ensemble.py

```diff
@@ -125,15 +172,32 @@ def reset(self):
                 self._N[t] = None
             if 'mu_{}'.format(t) in self.conjugates:
                 self._mu[t] = None
-        self.rdf = core.PairMatrix(self.types)
+        self.rdf = core.PairMatrix(types=self.types)
 
         return self
 
     def copy(self):
+        """Copies the ensemble and returns the copy with reset parameters.
+
+        Returns
+        -------
+        :py:class:`Ensemble`
+            A copy of the ensemble object with reset parameters.
+
+        """
         ens = copy.deepcopy(self)
         return ens.reset()
 
     def save(self, basename='ensemble'):
```

Comment:
We should revisit if there is a better way to save this other than making multiple files. Just making a note for now.

Reply:
Noting here: as we discussed, we will dump all the data into one JSON file.